build.sc

```diff
@@ -31,7 +31,7 @@ class KubernetesClientModule(val crossScalaVersion: String)
   object test extends Tests {
     def testFrameworks    = Seq("munit.Framework")
     override def forkArgs = super.forkArgs() :+ "-Djdk.tls.client.protocols=TLSv1.2"
-    override def ivyDeps  = super.ivyDeps() ++ Agg(ivy"org.scalameta::munit:0.7.26")
+    override def ivyDeps  = super.ivyDeps() ++ Agg(ivy"org.scalameta::munit:0.7.26") ++ logback
```

Comment:
add for logging in tests for debugging purpose

Reply:
Do we need to add logback? Adding log4cats is not enough?

Reply:
log4cats is only a logging facade. Actual logging framework should be added write the logs. 
logback is only added for Integration tests to be able to print project or other dependencies like http4s logging statements. Sometimes logging of an HTTP request is needed for tests debugging purposes.